dashboard/dashboard.py

```diff
@@ -2,6 +2,81 @@
 
 import re
 import streamlit as st
+import psycopg2
+from os import environ
+from dotenv import load_dotenv
+from psycopg2.extras import RealDictCursor
+
+
+def get_connection() -> psycopg2.connect:
+    """Returns connection to RDS."""
+    return psycopg2.connect(
+        dbname=environ.get("DB_NAME"),
+        host=environ.get("DB_HOST"),
+        user=environ.get("DB_USERNAME"),
+        password=environ.get("DB_PASSWORD"),
+        port=environ.get("DB_PORT")
+    )
+
+
+def get_cursor(conn) -> RealDictCursor:
```

Comment:
Is it possible to give the connections a type hint like you did above?

Reply:
good spot, just added any missing type hints.